data/campaigns/Heir_To_The_Throne/overworld/00a_western_wesnoth.cfg

```diff
@@ -0,0 +1,1625 @@
+#textdomain wesnoth-h2tt
+#wmlindent: start ignoring
+
+#######################################################################################################################################################
+#                                                                        FENCE HINT
+#######################################################################################################################################################
+# Give the player a way to bypass the 3 opening scenarios if they want to. This is also a nice little reward for exploration/interaction.
+# I'd like breaking the fence to be a variant for experienced players who want a little more variety, not a meta must-do option that's clearly stronger than the default 3 scenarios.
+[event]
+    name=prestart
+    {FILTER_CONDITION( {VARIABLE_CONDITIONAL bm_found_fence not_equals yes} )}
+
+    {NAMED_UNIT 2 "Elvish Civilian" 1 59 fence_wanderer _"Wanderer" ()} {FACING sw} {GENDER male}
+
+    [event]
+        name=moveto
+        first_time_only=no
+        {FILTER( {FILTER_ADJACENT id=fence_wanderer} )}
+        [message]
+            speaker=fence_wanderer
+            message= _ "Hello, young prince. A lovely day for wandering, isn’t it?"
+        [/message]
+        [message]
+            speaker=fence_wanderer
+            message= _ "I once wandered my way past a loose board set in the fence between the wood and the bay. I’ve forgetten exactly where, but I’m sure you could find it if you wandered back and forth, back and forth enough times."
```

Comment:
forgotten

Also, the repeated 'back and forth' doesn't make sense to me, unless you really wanted to reinforce that's what the player should do.